.circleci/config.yml

```diff
@@ -5,16 +5,28 @@ jobs:
 
     docker:
         - image: gcc:5.4.0 
-
+    
     steps:
 
       - checkout
 
       - run:
           name: update common
-          command: git submodule update
+          command: git submodule update --init
+
+      - run:
+          name: apt update
+          command: apt update
```

Comment:
I think you could just do `- run: apt update` since the separate name and command do not really help here.